### DISCUSSION TITLE
Run v8 and old versions side-by-side

### DISCUSSION THREAD

**Discussion by dcworldwide** [TanStack/table#4040] [Q&A] [answered]
We are using react-table v6.

Will it be possible to have v8 and v6 installed in the same project, to allow us to progressively migrate tables?

**Comment by tannerlinsley** [comment, marked as answer]
Yes, they use different package names.

**Reply by dcworldwide** [reply]
Thanks Tanner, I see...

react-table = old 
@tanstack/react-table = new
